### PR TITLE
Update Bit

### PR DESCRIPTION
**Pull request by adamrusted**
bit

**Issue:** contibutes toward #9072

**Similarweb rank:** [461,793](https://www.similarweb.com/website/bit.dev/#overview)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
New icon extracted from the font used in the footer.

SVG code as follows:

```html
<svg viewBox="0 0 2580 1252.4439" xmlns="http://www.w3.org/2000/svg"><path d="M458 550.0008q0 37-21 58.5-21 21.5-60 21.5-14 0-26.5-3-12.5-3-22.5-8-12-7-13.5-15-1.5-8-1.5-16l0-168q0-9 5.5-13.5 5.5-4.5 16.5-4.5 11 0 16.5 4.5 5.5 4.5 5.5 13.5l0 64q6-4 14.5-6.5 8.5-2.5 17.5-2.5 30 0 49.5 19.5 19.5 19.5 19.5 55.5l0 0z m-101 39q4 3 9.5 4 5.5 1 10.5 1 17 0 26-10.5 9-10.5 9-30.5 0-20-9-31-9-11-25-11-6 0-11.5 2-5.5 2-9.5 5l0 71 0 0z m184-157q0 11-7.5 18.5-7.5 7.5-18.5 7.5-12 0-19.5-7.5-7.5-7.5-7.5-18.5 0-12 7.5-19 7.5-7 19.5-7 11 0 18.5 7 7.5 7 7.5 19z m-4 61l0 118q0 9-5.5 13.5-5.5 4.5-16.5 4.5-11 0-17-4.5-6-4.5-6-13.5l0-118q0-9 6-13.5 6-4.5 17-4.5 11 0 16.5 4.5 5.5 4.5 5.5 13.5z m135 96q9 0 12.5 7.5 3.5 7.5 3.5 13.5 0 3-2 6.5-2 3.5-6 6.5-6 3-15 5-9 2-18 2-18 0-29-6.5-11-6.5-17-16.5-5-9-6-19.5-1-10.5-1-22.5l0-49-12 0q-9 0-13.5-5-4.5-5-4.5-14 0-9 4.5-14 4.5-5 13.5-5l12 0 0-29q0-9 5.5-13.5 5.5-4.5 16.5-4.5 11 0 16.5 4.5 5.5 4.5 5.5 13.5l0 29 28 0q9 0 13.5 5 4.5 5 4.5 14 0 9-4.5 14-4.5 5-13.5 5l-28 0 0 49q0 5 0.5 10.5 0.5 5.5 2.5 10.5 2 3 5 4.5 3 1.5 8 1.5 2 0 5-0.5 3-0.5 5-0.5 2-1 4-1.5 2-0.5 4-0.5l0 0z m-666-78q0-103 39-193 39-90.00000000000003 106-157.00000000000003 67-67 157-106 90-39 192-39 102 3.552713678800501e-15 192 39 90 39 157.5 106 67.5 67 105.5 157.00000000000003 39 90 39 193 0 102-39 192-38 90-105.5 157-67.5 67-157.5 106-90 39-192 39-102 0-192-39-90-39-157-106-67-67-106-157-39-90-39-192l0 0z m494-459q-95-2.842170943040401e-14-178 35.99999999999997-84 36-146 98.5-62 62.5-98 145.50000000000003-36 84-36 179 0 95 36 178 36 84 98 146 62 62 146 98 83 36 178 36 95 0 179-36 83-36 145-98 62-62 98-146 37-83 37-178 0-95-37-179-36-83.00000000000003-98-145.50000000000003-62-62.5-145-98.5-84-36-179-36l0 0z m-149 104.99999999999997q3 7 1 14-2 7-9 11-57 30-96.5 66.5-39.5 36.50000000000003-71.5 96.50000000000003-4 7-11 9-7 2-13-1-7-4-9-11-2-7 1-13 35-66 78.5-106.50000000000003 43.5-40.5 104.5-72.5 7-4 14-1.5 7 2.5 11 8.5l0 0z"/></svg>
```

### CODE REVIEW

File: icons/bit.svg [hunk 1]
```diff
@@ -1 +1 @@
...
....316-3.23.947-4.688s1.49-2.729 2.575-3.814S5.88 1.567 7.336.935 10.348-.012 12-.012s3.207.316 4.664.947A12.339 12.339 0 0 1 20.49 3.51a11.76 11.76 0 0 1 2.563 3.814c.631 1.457.947 3.02.947 4.688a11.61 11.61 0 0 1-.947 4.664 11.759 11.759 0 0 1-2.563 3.814 12.339 12.339 0 0 1-3.826 2.575c-1.458.631-3.012.947-4.664.947s-3.207-.316-4.664-.947-2.729-1.49-3.814-2.575C2.437 19.405 1.58 18.134.947 16.676S0 13.664 0 12.012zM12 .862c-1.538 0-2.98.292-4.324.875-1.36.583-2.542 1.38-3.547 2.392S2.333 6.32 1.75 7.664c-.583 1.36-.875 2.81-.875 4.348s.292 2.98.875 4.324a11.197 11.197 0 0 0 2.38 3.547 11.196 11.196 0 0 0 3.547 2.38c1.344.583 2.786.875 4.324.875s2.988-.292 4.348-.875a11.272 11.272 0 0 0 3.523-2.38 11.196 11.196 0 0 0 2.38-3.547c.6-1.344.899-2.785.899-4.324s-.3-2.988-.899-4.348c-.583-1.344-1.377-2.522-2.38-3.534s-2.179-1.81-3.523-2.393A10.918 10.918 0 0 0 12 .862zM8.38 3.413a.496.496 0 0 1 .025.34.442.442 0 0 1-.219.267c-.923.486-1.704 1.025-2.344 1.616S4.624 7.008 4.105 7.98a.443.443 0 0 1-.267.218.423.423 0 0 1-.316-.024.443.443 0 0 1-.218-.267.424.424 0 0 1 .024-.316C3.895 6.522 4.53 5.66 5.235 5.004s1.55-1.243 2.538-1.761a.392.392 0 0 1 .34-.036c.114.04.203.109.268.206z"/></svg>
```
Comment (service-paradis) [inline icons/bit.svg:1]: I see some [small differences](image).
Can you take another look to be sure?

**Review by service-paradis — CHANGES_REQUESTED** [review]

File: icons/bit.svg (hunk 1 above)
Comment (adamrusted) [reply icons/bit.svg:1]: Which version are you comparing against? There's a colour variation in the header, but this monochrome version was pulled from the footer (it's a font file)

Comment (service-paradis) [reply icons/bit.svg:1]: I used the SVG from the font file. I can take another look to be sure.

Comment (adamrusted) [reply icons/bit.svg:1]: I'll try and re-generate from the font file. Bear with while I re-surface the file 🙃

**Review by service-paradis — APPROVED** [review]
LGTM now 🎉 Thank you @adamrusted!

**Pull request merged**